Test T2 sites for goodIO list

**Impact of the bug**
T2 site utilization

**Describe the bug**
So far, we were following this subject over Slack and other channels. Here is a late and very necessary issue for this matter:

Workflows which read classical mixing PU are assigned to sites whose local storage capacities are good enough to handle these requests. These sites are maintained in a static list [1] and this list is quite old and it is required to be updated according to the up-to-date site capacities.

Here are the sites to be tested for this subject:
```
T2_BE_UCL,T2_BR_SPRACE,T2_CN_Beijing,T2_GR_Ioannina,T2_HU_Budapest,T2_IN_TIFR,T2_KR_KISTI,T2_RU_IHEP,T2_RU_INR,T2_RU_JINR,T2_TR_METU,T2_TW_NCHC,T2_UK_SGrid_Bristol,T2_UK_SGrid_RALPP,T2_US_Vanderbilt,T2_BR_UERJ,T2_FI_HIP,T2_UA_KIPT
```

So far, we have transferred the 5% of the following classical mixing PU to all of these sites [2]
```
/MinBias_TuneCP5_13TeV-pythia8/RunIISummer19UL18SIM-106X_upgrade2018_realistic_v4-v1/SIM
```

We have run multiple test workflows by using some hacks to read the partial copies [3], but all of these were failed to read the partial copies at the sites: [4]
```
Fatal Exception (Exit code: 8021)
An exception of category 'FileReadError' occurred while
[0] Processing Event run: 1 lumi: 188156 event: 18815510 stream: 0
[1] Running path 'PREMIXoutput_step'
[2] Prefetching for module PoolOutputModule/'PREMIXoutput'
[3] Calling method for module MixingModule/'mix'
[4] Reading branch EventAuxiliary
[5] Calling XrdFile::readv()
[6] XrdAdaptor::ClientRequest::HandleResponse() failure while running connection recovery
[7] Handling XrdAdaptor::RequestManager::requestFailure()
[8] In XrdAdaptor::RequestManager::OpenHandler::HandleResponseWithHosts()
Exception Message:
XrdCl::File::Open(name='root://grid143.kfki.hu//store/mc/RunIISummer19UL18SIM/MinBias_TuneCP5_13TeV-pythia8/SIM/106X_upgrade2018_realistic_v4-v1/70004/2C326D3C-4B13-4242-B155-EA9B819C33DA.root', flags=0x10, permissions=0660) => error '[FATAL] Redirect limit has been reached' (errno=0, code=306)
Additional Info:
[a] Original error: '[ERROR] Operation expired' (errno=0, code=206, source=ceph-gw10.gridpp.rl.ac.uk:1094 (site T1_UK_RAL)).
[b] Original failed source is ceph-gw10.gridpp.rl.ac.uk:1094 (site T1_UK_RAL)
[c] Disabled source: ceph-gw10.gridpp.rl.ac.uk:1094
```

[1] https://github.com/CMSCompOps/WmAgentScripts/blob/master/unifiedConfiguration.json#L99
[2] https://its.cern.ch/jira/browse/CMSTRANSF-187
[3] https://github.com/dmwm/WMCore/pull/10051
[4] https://cms-unified.web.cern.ch/cms-unified/report/haozturk_task_PPD-RunIISummer19ULPrePremix-Backfill-00004__v1_T_201105_105839_7372



**How to reproduce it**
Resubmit a test workflow to the backfill agent with secondary xrootd enabled. Make sure that @amaltaro applied the hack to the backfill agent. Following workflow is the suitable candidate for the test: https://cmsweb.cern.ch/reqmgr2/fetch?rid=haozturk_task_PPD-RunIISummer19ULPrePremix-Backfill-00004__v1_T_201105_105839_7372

**Expected behavior**
Workflow should run at the sites and be able to read the partial copies at the sites. Then, we can decide in which sites the jobs are successful to read the partial copies

**Additional context and error message**
@amaltaro @z4027163  FYI

--- a/unifiedConfiguration.json
+++ b/unifiedConfiguration.json
@@ -97,7 +97,7 @@
    "description" : "Over-ride the available space at a phedex end point"
  },
  "sites_with_goodIO": {
-    "value" : ["T2_CH_CSCS","T2_DE_DESY","T2_DE_RWTH","T2_ES_CIEMAT","T2_FR_GRIF_LLR", "T2_FR_GRIF_IRFU", "T2_FR_IPHC","T2_FR_CCIN2P3","T2_IT_Bari", "T2_IT_Legnaro", "T2_IT_Pisa", "T2_IT_Rome","T2_UK_London_Brunel", "T2_UK_London_IC","T2_US_Caltech","T2_US_MIT","T2_US_Nebraska","T2_US_Purdue","T2_US_UCSD","T2_US_Wisconsin","T2_US_Florida","T2_BE_IIHE","T2_EE_Estonia","T2_PL_Swierk","T2_CH_CERN","T2_CH_CERN_HLT","T2_KR_KISTI","T2_UK_SGrid_RALPP","T2_BR_SPRACE","T2_UA_KIPT"],
+    "value" : ["T2_CH_CSCS","T2_DE_DESY","T2_DE_RWTH","T2_ES_CIEMAT","T2_FR_GRIF_LLR", "T2_FR_GRIF_IRFU", "T2_FR_IPHC","T2_FR_CCIN2P3","T2_IT_Bari", "T2_IT_Legnaro", "T2_IT_Pisa", "T2_IT_Rome","T2_UK_London_Brunel", "T2_UK_London_IC","T2_US_Caltech","T2_US_MIT","T2_US_Nebraska","T2_US_Purdue","T2_US_UCSD","T2_US_Wisconsin","T2_US_Florida","T2_BE_IIHE","T2_EE_Estonia","T2_PL_Swierk","T2_CH_CERN","T2_CH_CERN_HLT","T2_KR_KISTI","T2_UK_SGrid_RALPP","T2_BR_SPRACE","T2_UA_KIPT","T2_BR_UERJ","T2_BE_UCL","T2_RU_JINR"],
     "description" : "The sites identified as having good storage bandwidth"
  },
  "sites_with_goodAAA": {

--- a/unifiedConfiguration.json
+++ b/unifiedConfiguration.json
@@ -97,7 +97,7 @@
    "description" : "Over-ride the available space at a phedex end point"
  },
  "sites_with_goodIO": {
-    "value" : ["T2_CH_CSCS","T2_DE_DESY","T2_DE_RWTH","T2_ES_CIEMAT","T2_FR_GRIF_LLR", "T2_FR_GRIF_IRFU", "T2_FR_IPHC","T2_FR_CCIN2P3","T2_IT_Bari", "T2_IT_Legnaro", "T2_IT_Pisa", "T2_IT_Rome","T2_UK_London_Brunel", "T2_UK_London_IC","T2_US_Caltech","T2_US_MIT","T2_US_Nebraska","T2_US_Purdue","T2_US_UCSD","T2_US_Wisconsin","T2_US_Florida","T2_BE_IIHE","T2_EE_Estonia","T2_PL_Swierk","T2_CH_CERN","T2_CH_CERN_HLT"],
+    "value" : ["T2_CH_CSCS","T2_DE_DESY","T2_DE_RWTH","T2_ES_CIEMAT","T2_FR_GRIF_LLR", "T2_FR_GRIF_IRFU", "T2_FR_IPHC","T2_FR_CCIN2P3","T2_IT_Bari", "T2_IT_Legnaro", "T2_IT_Pisa", "T2_IT_Rome","T2_UK_London_Brunel", "T2_UK_London_IC","T2_US_Caltech","T2_US_MIT","T2_US_Nebraska","T2_US_Purdue","T2_US_UCSD","T2_US_Wisconsin","T2_US_Florida","T2_BE_IIHE","T2_EE_Estonia","T2_PL_Swierk","T2_CH_CERN","T2_CH_CERN_HLT","T2_KR_KISTI","T2_UK_SGrid_RALPP","T2_BR_SPRACE","T2_UA_KIPT"],
     "description" : "The sites identified as having good storage bandwidth"
  },
  "sites_with_goodAAA": {

